userspace/libscap/engine/bpf/scap_bpf.c

```diff
@@ -547,7 +552,22 @@ static int32_t load_tracepoint(struct bpf_engine* handle, const char *event, str
 		return SCAP_FAILURE;
 	}
 
-	fd = bpf_load_program(prog, program_type, insns_cnt, error, BPF_LOG_SIZE);
+	/* 'event' looks like "raw_tracepoint/raw_syscalls/sys_enter". Skip
+	 * two '/' to find the last word, if possible.
+	 */
+	prog_name = strstr(event, "/");
```

Comment:
I'd use `strrchr` instead, that automatically fetches last occurrence of '/'.  
Then, if anything is found, we can use just `prog_name++`.